### ISSUE TITLE
It would be nice to have a quick-search or filter box

### ISSUE THREAD

**Issue by Brianetta** [Sneezry/authenticator#273]
I currently have 28 active accounts, and likely more in the future. Scrolling through the list, reading the accounts until I find the one I need to use, is something that the computer could help me with. I'd like ot be able to type in a few letters, and for the app to filter out those without those letters in their account name.

**Comment by mymindstorm** [comment]
There is a way to make a code go to the top depending on what site you visit.. See: #101

**Comment by mymindstorm** [comment]
If you haven't already you should take a backup. There are old bugs about having too many codes causing problems, but I'm not sure if that still applies to the latest version.

**Comment by Brianetta** [comment]
Yes, I have my most common at the top and at the bottom, but I do use all of them fairly frequently.

I maintain backups. I've been using this app for years, and have to restore the database every few months when it somehow manages to truncate the stored JSON and fails to decrypt anything. Part of my workflow when adding a new account is to decrypt, back it up, re-encrypt and then separately encrypt the backed up file.

**Comment by mymindstorm** [comment]
Could you run
```javascript
chrome.storage.sync.getBytesInUse(bytes=>console.log(bytes))
```
in the console (right click pop-up and hit inspect) and tell me the number it says?

**Comment by Brianetta** [comment]
It says: `VM69:1 Uncaught TypeError: Cannot read property 'sync' of undefined
    at <anonymous>:1:16`

It usually syncs everything. Would this be better as a separate issue?

**Comment by mymindstorm** [comment]
Due to scope it won't let you check the storage space from anywhere but a console launched from the extension. Try going to `chrome://extensions` and click the "Inspect views: background page" link (You have to check developer options at the top right to see the link)
![image](https://user-images.githubusercontent.com/27789806/35936146-093f2fda-0c08-11e8-8af4-8a773add39ec.png)
